docs/architecture/ADRs/0009_review_process.md

```diff
@@ -0,0 +1,22 @@
+# Data Review Process
+
+Date - 18/10/22
+
+## Status
+In progress
+
+## Context
+When new data is uploaded to the site, it needs to be reviewed by a member of the Vector Atlas team before it is made publicly available.
+
+We need to formalise a process for the review of data uploaded to the site.
+
+## Decision
+The review process is shown in the image below:
+![data review process](./images/review_process.png)
+
+Additional comments from review:
+ - The dataset will be read-only once approved
+ - The templates will be available from the site before upload
+
+## Consequences
+New stories have been written covering technical implementation of this process.
```

Comment:
Can you describe the technical approach here in the consequences, mainly to cover what we've decided about managing the in-review data and how we keep it separate or how we differentiate data waiting to be published.